### PR TITLE
add clean command

### PR DESCRIPTION
**Pull request by mordamax**
Fixes: #158 
`bot clean` with 👀 and 👍 reaction on start and finish accordingly, cuz it could take time to do that, and it needs to be "responsive" and more transparent of what is going on.
I was trying to set "common" way to react on commands, but whole PR looked over😵‍💫smiled, so keeping only for `bot clean`  where it makes more sense

image

### CODE REVIEW

File: src/bot/types.ts [hunk 1]
```diff
@@ -22,12 +22,27 @@ export type WebhookEventPayload<E extends WebhookEvents> = E extends "issue_comm
   ? IssueCommentCreatedEvent
   : never
 
+export class PullRequestError {
+  constructor(
+    public pr: PullRequestData,
+    public comment: {
+      body: string
+      botCommentId?: number
+      requesterCommendId: number
```
Comment (rzadp) [inline src/bot/types.ts:31]:
```suggestion
      requesterCommentId: number
```

**Review by rzadp — APPROVED** [review]
Looking good to me.

File: src/github.ts [hunk 2]
```diff
@@ -222,6 +222,101 @@ export const updateComment = async (
...
+  ...args: Parameters<typeof octokit.reactions.deleteForIssueComment>
+): Promise<void> => {
+  const response = await octokit.reactions.deleteForIssueComment(...args)
+
+  if (response.status !== 204) {
+    ctx.logger.error(response, `reactToComment response unsuccessful`)
+  }
+}
+
+export const cleanComments = async (
```
Comment (mutantcornholio) [inline src/github.ts:251]: You could remove most of the function body, if you would use `opstooling-integrations`, which has `makePaginateMethod` helper. Just saying.

File: src/bot/types.ts (hunk 1 above)
Comment (mordamax) [reply src/bot/types.ts]: [256277e](https://github.com/paritytech/command-bot/pull/157/commits/256277e5bfa4968ee73a7a931320f19a46c3d0da)

Comment (mordamax) [reply src/bot/types.ts]: [209f503](https://github.com/paritytech/command-bot/pull/157/commits/209f5038599de4604bd8cc6f1847f954adc89100) 😅

**Pull request merged**